[08/Dec/2014:15:48:01] ENGINE Bus STARTING
[08/Dec/2014:15:48:01] ENGINE Started monitor thread 'Monitor'.
[08/Dec/2014:15:48:01] ENGINE Started monitor thread '_TimeoutMonitor'.
[08/Dec/2014:15:48:01] ENGINE Serving on http://0.0.0.0:8989
[08/Dec/2014:15:48:01] ENGINE Bus STARTED
[08/Dec/2014:15:53:01] ENGINE Error in background task thread function <bound method Daemon.execute of <controller.daemon.Daemon object at 0x0000000002815D68>>.
Traceback (most recent call last):
  File "C:\Python34\lib\site-packages\cherrypy\process\plugins.py", line 500, in run
    self.function(*self.args, **self.kwargs)
  File "C:\Users\vbalaud\Documents\memdash\controller\daemon.py", line 21, in execute
    for server in MDatabase.get_servers():
AttributeError: type object 'MDatabase' has no attribute 'get_servers'

[08/Dec/2014:15:54:16] ENGINE Bus STOPPING
[08/Dec/2014:15:54:16] ENGINE HTTP Server cherrypy._cpwsgi_server.CPWSGIServer(('0.0.0.0', 8989)) shut down
[08/Dec/2014:15:54:16] ENGINE Stopped thread 'Monitor'.
[08/Dec/2014:15:54:16] ENGINE Stopped thread '_TimeoutMonitor'.
[08/Dec/2014:15:54:16] ENGINE Bus STOPPED
[08/Dec/2014:15:54:16] ENGINE Bus EXITING
[08/Dec/2014:15:54:16] ENGINE Bus EXITED
[08/Dec/2014:15:54:16] ENGINE Waiting for child threads to terminate...
[08/Dec/2014:15:54:20] ENGINE Bus STARTING
[08/Dec/2014:15:54:20] ENGINE Started monitor thread '_TimeoutMonitor'.
[08/Dec/2014:15:54:20] ENGINE Started monitor thread 'Monitor'.
[08/Dec/2014:15:54:20] ENGINE Serving on http://0.0.0.0:8989
[08/Dec/2014:15:54:20] ENGINE Bus STARTED
[08/Dec/2014:15:59:20] ENGINE Error in background task thread function <bound method Daemon.execute of <controller.daemon.Daemon object at 0x0000000002FF5B70>>.
Traceback (most recent call last):
  File "C:\Python34\lib\site-packages\cherrypy\process\plugins.py", line 500, in run
    self.function(*self.args, **self.kwargs)
  File "C:\Users\vbalaud\Documents\memdash\controller\daemon.py", line 21, in execute
    for server in MDatabase.get_servers():
AttributeError: type object 'MDatabase' has no attribute 'get_servers'

[08/Dec/2014:16:18:46] ENGINE Bus STOPPING
[08/Dec/2014:16:18:46] ENGINE HTTP Server cherrypy._cpwsgi_server.CPWSGIServer(('0.0.0.0', 8989)) shut down
[08/Dec/2014:16:18:46] ENGINE Stopped thread 'Monitor'.
[08/Dec/2014:16:18:46] ENGINE Stopped thread '_TimeoutMonitor'.
[08/Dec/2014:16:18:46] ENGINE Bus STOPPED
[08/Dec/2014:16:18:46] ENGINE Bus EXITING
[08/Dec/2014:16:18:46] ENGINE Waiting for child threads to terminate...
[08/Dec/2014:16:18:46] ENGINE Bus EXITED
[08/Dec/2014:16:18:50] ENGINE Bus STARTING
[08/Dec/2014:16:18:50] ENGINE Started monitor thread 'Monitor'.
[08/Dec/2014:16:18:50] ENGINE Started monitor thread '_TimeoutMonitor'.
[08/Dec/2014:16:18:50] ENGINE Serving on http://0.0.0.0:8989
[08/Dec/2014:16:18:50] ENGINE Bus STARTED
[08/Dec/2014:16:20:14] ENGINE Bus STOPPING
[08/Dec/2014:16:20:14] ENGINE HTTP Server cherrypy._cpwsgi_server.CPWSGIServer(('0.0.0.0', 8989)) shut down
[08/Dec/2014:16:20:14] ENGINE Stopped thread '_TimeoutMonitor'.
[08/Dec/2014:16:20:14] ENGINE Stopped thread 'Monitor'.
[08/Dec/2014:16:20:14] ENGINE Bus STOPPED
[08/Dec/2014:16:20:14] ENGINE Bus EXITING
[08/Dec/2014:16:20:14] ENGINE Waiting for child threads to terminate...
[08/Dec/2014:16:20:14] ENGINE Bus EXITED
[08/Dec/2014:16:20:21] ENGINE Bus STARTING
[08/Dec/2014:16:20:21] ENGINE Started monitor thread 'Monitor'.
[08/Dec/2014:16:20:21] ENGINE Started monitor thread '_TimeoutMonitor'.
[08/Dec/2014:16:20:21] ENGINE Serving on http://0.0.0.0:8989
[08/Dec/2014:16:20:21] ENGINE Bus STARTED
[08/Dec/2014:16:25:21] ENGINE Error in background task thread function <bound method Daemon.execute of <controller.daemon.Daemon object at 0x00000000026F92B0>>.
Traceback (most recent call last):
  File "C:\Python34\lib\site-packages\cherrypy\process\plugins.py", line 500, in run
    self.function(*self.args, **self.kwargs)
  File "C:\Users\vbalaud\Documents\memdash\controller\daemon.py", line 21, in execute
    for server in MDatabase.get_servers():
AttributeError: type object 'MDatabase' has no attribute 'get_servers'

[08/Dec/2014:16:32:04] ENGINE Bus STOPPING
[08/Dec/2014:16:32:04] ENGINE HTTP Server cherrypy._cpwsgi_server.CPWSGIServer(('0.0.0.0', 8989)) shut down
[08/Dec/2014:16:32:04] ENGINE Stopped thread '_TimeoutMonitor'.
[08/Dec/2014:16:32:04] ENGINE Stopped thread 'Monitor'.
[08/Dec/2014:16:32:04] ENGINE Bus STOPPED
[08/Dec/2014:16:32:04] ENGINE Bus EXITING
[08/Dec/2014:16:32:04] ENGINE Waiting for child threads to terminate...
[08/Dec/2014:16:32:04] ENGINE Bus EXITED
[08/Dec/2014:16:32:08] ENGINE Bus STARTING
[08/Dec/2014:16:32:08] ENGINE Started monitor thread 'Monitor'.
[08/Dec/2014:16:32:08] ENGINE Started monitor thread '_TimeoutMonitor'.
[08/Dec/2014:16:32:08] ENGINE Serving on http://0.0.0.0:8989
[08/Dec/2014:16:32:08] ENGINE Bus STARTED
[08/Dec/2014:16:37:08] ENGINE Error in background task thread function <bound method Daemon.execute of <controller.daemon.Daemon object at 0x0000000003026CC0>>.
Traceback (most recent call last):
  File "C:\Python34\lib\site-packages\cherrypy\process\plugins.py", line 500, in run
    self.function(*self.args, **self.kwargs)
  File "C:\Users\vbalaud\Documents\memdash\controller\daemon.py", line 21, in execute
    for server in MDatabase.get_servers():
AttributeError: type object 'MDatabase' has no attribute 'get_servers'

[08/Dec/2014:16:39:29] ENGINE Bus STOPPING
[08/Dec/2014:16:39:29] ENGINE HTTP Server cherrypy._cpwsgi_server.CPWSGIServer(('0.0.0.0', 8989)) shut down
[08/Dec/2014:16:39:29] ENGINE Stopped thread '_TimeoutMonitor'.
[08/Dec/2014:16:39:29] ENGINE Stopped thread 'Monitor'.
[08/Dec/2014:16:39:29] ENGINE Bus STOPPED
[08/Dec/2014:16:39:29] ENGINE Bus EXITING
[08/Dec/2014:16:39:29] ENGINE Waiting for child threads to terminate...
[08/Dec/2014:16:39:29] ENGINE Bus EXITED
[08/Dec/2014:16:39:32] ENGINE Bus STARTING
[08/Dec/2014:16:39:32] ENGINE Started monitor thread '_TimeoutMonitor'.
[08/Dec/2014:16:39:32] ENGINE Started monitor thread 'Monitor'.
[08/Dec/2014:16:39:32] ENGINE Serving on http://0.0.0.0:8989
[08/Dec/2014:16:39:32] ENGINE Bus STARTED
[08/Dec/2014:16:40:02] ENGINE Error in background task thread function <bound method Daemon.execute of <controller.daemon.Daemon object at 0x000000000287D438>>.
Traceback (most recent call last):
  File "C:\Python34\lib\site-packages\cherrypy\process\plugins.py", line 500, in run
    self.function(*self.args, **self.kwargs)
  File "C:\Users\vbalaud\Documents\memdash\controller\daemon.py", line 21, in execute
    for server in MDatabase.get_servers():
AttributeError: type object 'MDatabase' has no attribute 'get_servers'

[08/Dec/2014:16:43:46] ENGINE Bus STOPPING
[08/Dec/2014:16:43:46] ENGINE HTTP Server cherrypy._cpwsgi_server.CPWSGIServer(('0.0.0.0', 8989)) shut down
[08/Dec/2014:16:43:46] ENGINE Stopped thread 'Monitor'.
[08/Dec/2014:16:43:46] ENGINE Stopped thread '_TimeoutMonitor'.
[08/Dec/2014:16:43:46] ENGINE Bus STOPPED
[08/Dec/2014:16:43:46] ENGINE Bus EXITING
[08/Dec/2014:16:43:46] ENGINE Waiting for child threads to terminate...
[08/Dec/2014:16:43:46] ENGINE Bus EXITED
[08/Dec/2014:16:43:50] ENGINE Bus STARTING
[08/Dec/2014:16:43:50] ENGINE Started monitor thread '_TimeoutMonitor'.
[08/Dec/2014:16:43:50] ENGINE Started monitor thread 'Monitor'.
[08/Dec/2014:16:43:50] ENGINE Serving on http://0.0.0.0:8989
[08/Dec/2014:16:43:50] ENGINE Bus STARTED
[08/Dec/2014:16:44:20] ENGINE Error in background task thread function <bound method Daemon.execute of <controller.daemon.Daemon object at 0x0000000002837EF0>>.
Traceback (most recent call last):
  File "C:\Python34\lib\site-packages\cherrypy\process\plugins.py", line 500, in run
    self.function(*self.args, **self.kwargs)
  File "C:\Users\vbalaud\Documents\memdash\controller\daemon.py", line 21, in execute
    for server in MDatabase.get_servers():
AttributeError: type object 'MDatabase' has no attribute 'get_servers'

[08/Dec/2014:16:44:38] ENGINE Bus STOPPING
[08/Dec/2014:16:44:39] ENGINE HTTP Server cherrypy._cpwsgi_server.CPWSGIServer(('0.0.0.0', 8989)) shut down
[08/Dec/2014:16:44:39] ENGINE Stopped thread 'Monitor'.
[08/Dec/2014:16:44:39] ENGINE Stopped thread '_TimeoutMonitor'.
[08/Dec/2014:16:44:39] ENGINE Bus STOPPED
[08/Dec/2014:16:44:39] ENGINE Bus EXITING
[08/Dec/2014:16:44:39] ENGINE Waiting for child threads to terminate...
[08/Dec/2014:16:44:39] ENGINE Bus EXITED
[08/Dec/2014:16:44:43] ENGINE Bus STARTING
[08/Dec/2014:16:44:43] ENGINE Started monitor thread 'Monitor'.
[08/Dec/2014:16:44:43] ENGINE Started monitor thread '_TimeoutMonitor'.
[08/Dec/2014:16:44:43] ENGINE Serving on http://0.0.0.0:8989
[08/Dec/2014:16:44:43] ENGINE Bus STARTED
[08/Dec/2014:16:45:13] ENGINE Error in background task thread function <bound method Daemon.execute of <controller.daemon.Daemon object at 0x0000000002837DA0>>.
Traceback (most recent call last):
  File "C:\Python34\lib\site-packages\cherrypy\process\plugins.py", line 500, in run
    self.function(*self.args, **self.kwargs)
  File "C:\Users\vbalaud\Documents\memdash\controller\daemon.py", line 21, in execute
    for server in MDatabase.get_servers():
AttributeError: type object 'MDatabase' has no attribute 'get_servers'

[08/Dec/2014:16:45:27] ENGINE Bus STOPPING
[08/Dec/2014:16:45:27] ENGINE HTTP Server cherrypy._cpwsgi_server.CPWSGIServer(('0.0.0.0', 8989)) shut down
[08/Dec/2014:16:45:27] ENGINE Stopped thread '_TimeoutMonitor'.
[08/Dec/2014:16:45:27] ENGINE Stopped thread 'Monitor'.
[08/Dec/2014:16:45:27] ENGINE Bus STOPPED
[08/Dec/2014:16:45:27] ENGINE Bus EXITING
[08/Dec/2014:16:45:27] ENGINE Waiting for child threads to terminate...
[08/Dec/2014:16:45:27] ENGINE Bus EXITED
[08/Dec/2014:16:45:31] ENGINE Bus STARTING
[08/Dec/2014:16:45:31] ENGINE Started monitor thread 'Monitor'.
[08/Dec/2014:16:45:31] ENGINE Started monitor thread '_TimeoutMonitor'.
[08/Dec/2014:16:45:31] ENGINE Serving on http://0.0.0.0:8989
[08/Dec/2014:16:45:31] ENGINE Bus STARTED
[08/Dec/2014:16:46:01] ENGINE Error in background task thread function <bound method Daemon.execute of <controller.daemon.Daemon object at 0x0000000002877E10>>.
Traceback (most recent call last):
  File "C:\Python34\lib\site-packages\cherrypy\process\plugins.py", line 500, in run
    self.function(*self.args, **self.kwargs)
  File "C:\Users\vbalaud\Documents\memdash\controller\daemon.py", line 21, in execute
    for server in MDatabase.get_servers():
AttributeError: type object 'MDatabase' has no attribute 'get_servers'

